tools/travis/build.sh

```diff
@@ -26,7 +26,7 @@ cd $ROOTDIR/ansible
 
 $ANSIBLE_CMD wipe.yml
 $ANSIBLE_CMD openwhisk.yml
-$ANSIBLE_CMD postdeploy.yml
+$ANSIBLE_CMD postdeploy.yml -e catalog_source=catalog-repos
```

Comment:
This change should not be in this PR.
The final goal is to have the catalog content taken out from this repo, including catalog tests.
Travis CI for this repo would stop depending on catalog, testing the catalog content becomes a responsibility of the new repo "openwhisk-catalog"